### ISSUE TITLE
Patent / Copyright

### ISSUE THREAD

**Issue by jacobfriedman** [ridgeworks/clpBNR_pl#9]
Can you verify this is clear for publishing? As in, the patent expired 20 years ago on the method, or, the original spec docs (or implementation) haven't been copied verbatim, or, enough to get legal in a tussle? Thanks in advance!

**Comment by ridgeworks** [comment]
As manager of the group that did the original development .ca 1987-93 at Bell-Northern Reasearch (see <https://en.wikipedia.org/wiki/BNR_Prolog>), I'm unaware of any patent covering this material. In addition, there were a number of academic papers published (see <https://ridgeworks.github.io/BNRProlog-Papers/> ) which were placed in this repo for archival purposes with the consent of W. Older, one of the primary authors, and technical prime on the project.

Additional public domain material from a Carleton University graduate level course can be found here: <http://www.softwarepreservation.org/projects/prolog/bnr/doc/Older-Introduction_to_CLP%28BNR%29-1995.pdf/view>

I am the sole developer for this particular repo which uses none of the original code.

In summary, I'm not aware of any legal issue that would prevent you from using this software subject to the LICENSE conditions. I am interested in any feedback (good and bad) on applications that try to use it.

**Comment by jacobfriedman** [comment]
Alright! I actually gave them a call - BNR is a few doors down. I'll be closely watching & perhaps implementing this in the very near future for ISO8601 Prolog. Problems with clp over integers are literally unreasonable on an infinite (time) axis...

**Comment by ridgeworks** [comment]
Just to be clear, the BNR I worked for ceased to be a entity 25 years ago. At that time it became part of Nortel which went out of business in 2009. (One more reason why legal problems are very unlikely.)
